src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -196,6 +200,9 @@ const ChatInfoScreen = () => {
     loadingChannelInfo,
     isLoadingInitChat
   } = useChatInfoScreenHook();
+  const navigation = useNavigation();
+
+  const {onLeaveGroup, onReportGroup} = useGroupInfo(channelInfo?.id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useNavigation` hook is being used here to navigate between screens. Also, the `useGroupInfo` hook is being used to provide functionalities related to group information like leaving a group or reporting a group.
```diff
+   const navigation = useNavigation();
+   const {onLeaveGroup, onReportGroup} = useGroupInfo(channelInfo?.id);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->